.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,87 @@
+name: "Bug Report [NEW]"
+description: "Report an issue with the software."
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: "Summary"
+      description: "Provide a short description of the issue."
+      placeholder: "e.g., App crashes when opening a file"
```

```suggestion
      placeholder: "When playing a movie via the «Resume from» context menu, I get the error message «xyz»"
```
Placeholders should encourage descriptive responses